pxr/usd/usdSkel/skeletonQuery.cpp

```diff
@@ -356,7 +356,10 @@ UsdSkelSkeletonQuery::_ComputeSkinningTransforms(VtArray<Matrix4>* xforms,
 
         if (xforms->size() == inverseBindXforms.size()) {
             // xforms = inverseBindXforms * xforms
-            _MultTransforms<Matrix4>(inverseBindXforms, *xforms, *xforms);
+            // Explicit copy to avoid VT_LOG_STACK_ON_ARRAY_DETACH_COPY log
+            VtArray<Matrix4> xformsTmp(xforms->cbegin(), xforms->cend());
+            _MultTransforms<Matrix4>(inverseBindXforms, xformsTmp.AsConst(), xformsTmp);
```

Comment:
Could you please add the `TODO: MakeUnique()` comment here too, and remove the explicit copy?

Reply:
Sure, I'll remove the code I added. I didn't see @gitamohr previous comment last week. The code below is just copying the temporary result into xforms, it's not what @gitamohr is asking.